SWIPs/swip-message_to_honey.md

```diff
@@ -26,42 +25,75 @@ This SWIP is part of a series of SWIPs (but can be implemented on it's own). To
 
 ## Abstract
 <!--A short (~200 word) description of the technical issue being addressed.-->
-To enable changing the relative prices of Swarm messages for all nodes simultaneously, Swarm will provide a `MsgToHoney` payment oracle, initially managed by Swarm stakeholders and developers. The `MsgToHoney` oracle receives a message type and returns a `(price,validUntil)` tuple. The `MsgToHoney` oracle lives on the Ethereum blockchain and is managed by a multi-signature contract of Swarm stakeholders and developers. Once nodes receive a message to account for, they will query their local cache for a valid price for this message. If valid, the price will be applied by the node, if not, the nodes will query the `MsgToHoney` oracle. The oracle provides a new price before the old price expires, to allow nodes to pre-load new prices before old ones expire. If the oracle fails to submit a new price, while the old price is expired, nodes will continue querying the `MsgToHoney` oracle at a set interval and apply the most-recent price known to them. 
+To enable changing the relative prices of Swarm messages for all nodes simultaneously, Swarm will provide a `MsgToHoney` payment oracle, initially managed by Swarm stakeholders and developers. The `MsgToHoney` oracle returns an object from which the most recent price for a particular message can be inferred. Once nodes receive a message to account for, they will query their local cache for a valid price for this message. The local cache is kept up-to-date at all times by the nodes. Any query to the oracle returns an object with a fixed `Time To Live (TTL)`. After the `TTL` expires, nodes will query the oracle again. The `MsgToHoney` oracle lives on the Ethereum blockchain and is managed by a governance structure of Swarm developers and stakeholders.
```

Comment:
`for a particular message type`

Reply:
What do you mean here, could you mention the part of the sentence where you would like `for a particular message type` to be inserted? Thanks!

Reply:
Changed!

Reply:
changed!